specification/ai/Azure.AI.Projects/evaluations/routes.tsp

```diff
@@ -51,4 +51,24 @@ interface Evaluations {
     },
     ResourceCreatedResponse<AgentEvaluation>
   >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Cancel an evaluation run by name")
+  @post
+  @Azure.Core.pollingOperation(Evaluations.cancel)
+  cancel is EvaluationsOperations.ResourceAction<
+    Evaluation,
+    {},
+    AcceptedResponse &
+      Evaluation & {
+        @Azure.Core.pollingLocation()
+        @doc("The location for monitoring the evaluation state.")
+        @TypeSpec.Http.header("operation-location")
+        operationLocation: string;
+      }
+  >;
+
+  @doc("Delete an evaluation run by name")
+  @Azure.Core.pollingOperation(Evaluations.delete)
```

Comment:
You can't poll on the location you just deleted since that will return a 404 once the delete is completed. You need a separate status monitor url.